src/store/modules/accounts/hdWallet.js

```diff
@@ -63,7 +67,7 @@ export default {
       const { secretKey } = (options?.fromAccount)
         ? getAccountByAddress(options.fromAccount)
         : activeAccount.value;
-      return sign(data, secretKey);
+      return sign(data, new Uint8Array(Buffer.from(secretKey, 'hex')));
```

Comment:
```suggestion
      return sign(data, Buffer.from(secretKey, 'hex'));
```
usually Buffer extends Uint8Array, so this conversion shouldn't be necessary